Makefile

```diff
@@ -20,7 +20,7 @@ deploy: $(TOOLBIN)/kubectl $(TOOLBIN)/helm
                --namespace $(KUBE_NAMESPACE) --wait --timeout 120s
 
 .PHONY: test
-test: export BLUEPRINT_NAMESPACE?=fybrik-blueprints
+test: export DATA_ACCESS_MODULE_NAMESPACE?=fybrik-blueprints
```

Comment:
I think that we should drop "data access" and just say modules. So `MODULES_NAMESPACE`, `.Values.modulesNamespace`, etc. would be my recommendation.